Unreal/DazStudioPlugin/DzUnrealAction.h

```diff
@@ -17,6 +17,7 @@ class DzUnrealAction : public DzRuntimePluginAction {
 
 protected:
 	 int Port;
+     int NonInteractiveMode;
```

Comment:
In order to create a Script-accessible Property, you need to use the Q_PROPERTY macro so that the Qt SDK can wrap the variable and methods so that Daz Script interpreter can access it.

Please see the example implementation here: 
https://github.com/danielbui78/UDTU/blob/5e4e6e398261b7bc5c6b7fd9310d5df6623a05d2/Unity/DazStudioPlugin/DzUnityAction.h#L20

Here is the API reference about Qt Properties and use of Q_PROPERTY macro:
https://doc.qt.io/archives/qt-4.8/properties.html